euphonic/readers/castep.py

```diff
@@ -54,14 +54,17 @@ def read_phonon_dos_data(
         freqs = np.empty((0, n_branches))
         mode_grads = np.empty((0, n_branches))
         while True:
```

Comment:
Are you allowed Walrus? this is an ideal opportunity.

Reply:
Not yet :cry: